content/en/cloud_cost_management/google_cloud.md

```diff
@@ -48,6 +48,7 @@ The <a href="https://cloud.google.com/billing/docs/how-to/export-data-bigquery-t
 
 {{< img src="cloud_cost/billing_export.png" alt="Google Cloud project and dataset info highlighted" >}}
 
+_Newly created BigQuery datasets will only contain the most recent two months of data._
```

Comment:
```suggestion
_Newly created BigQuery billing export datasets only contain the most recent two months of data. It can take a day or two for this data to backfill in BigQuery._
```